packages/web-app/src/components/Dropdown.tsx

```diff
@@ -16,45 +15,52 @@ const styles = (theme: SaladTheme) => ({
 
 interface Props extends WithStyles<typeof styles> {
   options?: OptionsOrGroups<any, any>
+  value?: string
   onChange?: (value?: any) => void
 }
 
-const customStyles = {
-  singleValue: (provided: any, state: any) => {
-    const opacity = state.isDisabled ? 0.5 : 1
-    const transition = 'opacity 300ms'
-    const color = '#DBF1C1'
+const _Dropdown: FC<Props> = ({ classes, options, value, onChange }) => {
+  const selectedValue = value
+  const selectedOption = selectedValue && options?.find((option) => option.value === selectedValue)
 
-    return { ...provided, opacity, transition, color }
-  },
-}
+  const customStyles = {
+    option: (styles: any) => {
+      return {
+        ...styles,
+        cursor: 'pointer',
+      }
+    },
+    singleValue: (provided: any, state: any) => {
+      const opacity = state.isDisabled ? 0.5 : 1
+      const transition = 'opacity 300ms'
+      const color = '#DBF1C1'
 
-class _Dropdown extends Component<Props> {
-  public override render(): ReactNode {
-    const { options, onChange, classes } = this.props
-
-    return (
-      <Select
-        className={classes.container}
-        options={options}
-        onChange={onChange}
-        defaultValue={options && options[0]}
-        styles={customStyles}
-        theme={(selectTheme) => ({
-          // TODO: Get the theme data using react-jss
-          ...selectTheme,
-          borderRadius: 0,
-          colors: {
-            ...selectTheme.colors,
-            neutral0: '#0A2133',
-            primary25: '#1F4F22',
-            primary: '#DBF1C1',
-            neutral20: '#DBF1C1',
-          },
-        })}
-      />
-    )
+      return { ...provided, opacity, transition, color }
+    },
   }
+
+  return (
+    <Select
+      className={classes.container}
+      value={selectedOption}
```

Comment:
Can we align naming to 
```suggestion
      value={selectedValue}
```